pkg/kubelet/image_manager_test.go

```diff
@@ -37,6 +37,7 @@ func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *container.Fa
 	return &realImageManager{
 		runtime:      fakeRuntime,
 		policy:       policy,
+		minAge:       0,
```

Comment:
@vishh Sorry, I didn't find any reference about the test friendly time object, could you please kindly paste the reference. Great thanks!